controller/src/main/java/io/pravega/controller/task/Stream/StreamMetadataTasks.java

```diff
@@ -236,7 +236,7 @@ public void initializeStreamWriters(final ClientFactory clientFactory,
             } catch (Exception e) {
                 log.error("error sending request to requeststream {}", e);
                 if (e instanceof ScaleOperationExceptions.ScaleRequestNotEnabledException) {
```

Comment:
Until writer is set, we cant post scale request to request stream. 
Scale is performed by picking the event from the request stream. 
So if streamMetadataTasks does not have a 'writer' for request stream initialized, it cannot post requests.